dotCMS/src/main/java/com/dotcms/ema/proxy/ProxyResponse.java

```diff
@@ -0,0 +1,59 @@
+package com.dotcms.ema.proxy;
+
+import org.apache.http.Header;
+import org.apache.http.ProtocolVersion;
+import org.apache.http.StatusLine;
+
+public class ProxyResponse {
+
+  /**
+   * The Response.
+   */
+  final private byte[] response;
+  final private StatusLine status;
+  final private int responseCode;
+  final private Header[] headers;
+
+  public ProxyResponse(StatusLine status, byte[] out, Header[] headers) {
+    this.response = out;
+    this.status = status;
+    this.responseCode = status.getStatusCode();
+    this.headers = headers;
+  }
+
+  public ProxyResponse(final int responseCode, byte[] out, Header[] headers) {
```

Comment:
Codacy found an issue: [Consider using varargs for methods or constructors which take an array the last parameter.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5922984)